src/test/java/io/gingersnapproject/testcontainers/BaseGingersnapResourceLifecycleManager.java

```diff
@@ -90,14 +92,16 @@ private Map<String, String> internalStart() {
             "gingersnap.database.password", database.getPassword()
       ));
 
+      boolean isOracle = Profiles.isProfileActive("oracle");
       for (String rule : rules) {
          CountDownLatch prev = AUTOMATIC_STARTED_ENGINES.put(CacheIdentifier.of(rule, URI.create(cacheManager.hotRodURI())), new CountDownLatch(1));
          if (prev != null) prev.countDown();
          properties.putAll(Map.of(
                "gingersnap.rule.%s.connector.schema".formatted(rule), "debezium",
-               "gingersnap.rule.%s.connector.table".formatted(rule), "customer",
-               "gingersnap.rule.%s.key-columns".formatted(rule), "fullname"
+               "gingersnap.rule.%s.connector.table".formatted(rule), isOracle ? "CUSTOMER" : "customer",
+               "gingersnap.rule.%s.key-columns".formatted(rule), isOracle ? "FULLNAME" : "fullname"
```

Comment:
We can avoid the need to do this by surrounding column names with quotes when creating the table definition. We take this approach in the [cache-manager](https://github.com/gingersnap-project/cache-manager/blob/main/manager/src/test/resources/oracle/oracle-setup.sql).

Reply:
No success with that :(

Creating the columns with double quotes means we have to use double quotes everywhere we refer to the column. Since the new tests operate on the DB, we would need to have specific queries for Oracle because the other DBs parse double quotes differently.

Reply:
Argh I see, thanks for trying.